### DISCUSSION TITLE
Rescue (!) needed after "misconfiguration" of TS011F_plug_1 in Dev console

### DISCUSSION THREAD

**Discussion by rsteininger** [Koenkk/zigbee2mqtt#12308] [Devices]
hi there, 
I think I have "screwed" up my zigbee socket TS011F_plug_1 when trying to reset the energy meter. 

I tried to follow the guide on how to reset the energy consumption back to 0kWh, as documented here:
https://www.zigbee2mqtt.io/devices/TS011F_plug_1.html#reset-energy
However, after entering:
Endpoint: 1 Cluster: 0x00 Command: 0
I was not able to "execute", so I added the {0} in payload and pressed "execute". 
And ever since then the socket is ... just short of "dead". 
It is flagged as "offline" but I can still operate it in the web ui as on/off. 
However, all power measurement (voltage, power, current, energy) have "disappeared".

I am able to "remove" the device from my zigbee2mqtt setup and re-join but the problem is consistently the same ... 

So, my desperate question is:
1) is the a factory reset function for this device and / or 
2) is there a way to "undo" / reverse the configuration step I performed above!? 

(and lastly, 3) where did I go wrong with my interpretation of energy reset command?)

THANKS to all the gurus who hopefully understand my self created problem and might have  pointer for me on how to fix it!

**Comment by kkossev** [comment]
> I am able to "remove" the device from my zigbee2mqtt setup and re-join but the problem is consistently the same ...

Did you manage to get the smart plug working again?

**Comment by rsteininger** [comment]
hi @kkossev 
No, I still have a mostly dead zigbee plug sitting around. 
I do have another identical model and with the right knowledge, I might be able to read out registers and "reconfigure" / revive my dead plug but I don't have the knowledge to figure out how! 
Thanks,
Robert

**Reply by WroJan** [reply]
Hey, have you got it working?

**Reply by rsteininger** [reply]
No, same here. The one I "reset" is still in the same condition ... I bought new ones ...
